install.sh

```diff
@@ -84,21 +84,18 @@ msg "Getting URL for latest Home Assistant disk image..."
 RELEASE_EXT=vmdk.gz
 URL=$(cat<<EOF | python3
 import requests
-url = 'https://api.github.com/repos/home-assistant/operating-system/releases'
+url = "https://api.github.com/repos/home-assistant/operating-system/releases"
 r = requests.get(url).json()
-if 'message' in r:
-  exit()
+if "message" in r:
+    exit()
 for release in r:
-  if release['prerelease']:
-    continue
-  for asset in release['assets']:
-    if asset['name'].endswith('$RELEASE_EXT'):
-      global image_url
-      image_url = asset['browser_download_url']
-      break
-  if 'image_url' in globals():
-    print(image_url)
-    break
+    if release["prerelease"]:
+        continue
+    for asset in release["assets"]:
+        if asset["name"].endswith("$RELEASE_EXT"):
+            image_url = asset["browser_download_url"]
+            print(image_url)
```

Comment:
Good change

... or just as well `print(asset["browser_download_url"])`?

But I think you should not change the indentation level.  If the original author prefers 2 spaces, then your change proposal should probably keep his preference.  (Maybe the same thing for the ' and " quotes...)

Reply:
EDIT: Actually PEP8 suggests 4 spaces, so if you want to say it should follow that (even if it is just an inline snippet here) then I suppose you could suggest it.\